render.d.ts

```diff
@@ -0,0 +1,47 @@
+export as namespace render
```

Comment:
What the convention do we have `render` or `hyperappRender`? See https://github.com/hyperapp/logger/issues/18
Looks like we agreed to use `window.hyperappRender` should this namespace be the same?

Reply:
I sent a PR about this addressed at #11